Create ui-c8y-1020-0-0-asset-selector-showing-by-default-the-selected-element-4776.md (#2080)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1020-0-0-asset-selector-showing-by-default-the-selected-element-4776.md

---
date: ""

> title: asset selector showing by default the selected element (#4776)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Asset selector shows the selected element by default
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YdSEScrEC
    label: Cockpit
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-55762
version: 1020.0.0
---

> asset selector showing by default the selected element (#4776)

Author: The Claude AI suggested to use the following body:
```suggestion
Previously, the asset selector did not show the currently selected element by default, requiring users to manually search for and select the desired asset each time. With this change, the asset selector now automatically displays the selected element when opened. This improvement streamlines the user workflow by eliminating the need to repeatedly locate and select the same asset, saving time and effort.
```

> Previously, the asset selector did not show the currently selected element by default, requiring users to manually search for and select the desired asset each time. With this change, the asset selector now automatically displays the selected element when opened. This improvement streamlines the user workflow by eliminating the need to repeatedly locate and select the same asset, saving time and effort.

Reviewer: ```suggestion
Previously, the asset selector did not show the currently selected element by default, requiring users to always manually search for and select the desired asset. With this change, the asset selector now automatically displays the selected element when opened. This improvement streamlines the user workflow by eliminating the need to repeatedly locate and select the same asset, saving time and effort.
```
